contracts/UFragmentsPolicy.sol

```diff
@@ -30,20 +30,23 @@ contract UFragmentsPolicy is Ownable {
     event LogRebase(
         uint256 indexed epoch,
         uint256 exchangeRate,
-        uint256 volume24hrs,
+        uint256 cpi,
         int256 requestedSupplyAdjustment
     );
 
     UFragments public uFrags;
-    IMarketOracle public marketOracle;
 
-    // If the current exchange rate is within this absolute distance from the target, no supply
+    // Cpi oracle provides the current CPI-U data as a 18 decimal fixed point number
+    IOracle public cpiOracle;
+
+    // Market oracle provides the token:USD exchange rate as a 18 decimal fixed point number
```

Comment:
There's standard notation around exchange rates we can use to decrease any confusion.
e.g.:
```
In the case of the Japanese yen, it's USD/JPY, or dollar to yen. An exchange rate of 100 would mean that 1 dollar equals 100 yen.
```
from [https://www.investopedia.com/terms/e/exchangerate.asp](https://www.investopedia.com/terms/e/exchangerate.asp)

So that should be `token/USD`? How about including this example:
An oracle value of 1.5e18 it would mean 1 Ample is trading for $1.50.

Reply:
Yep. `token/USD` makes sense. Added the example aswell.